scripts/logfetch/tail.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import logfetch_base
+import requests
+import time
+import threading
+from singularity_request import get_json_response
+
+
+TAIL_LOG_FORMAT = '{0}/sandbox/{1}/read'
+READ_INTERVAL = 5
+
+def tail_logs(args):
+  if args.requestId:
+    sys.stderr.write('Fetching tasks\n')
+    tasks = [str(t) for t in logfetch_base.tasks_for_request(args)]
+  else:
+    tasks = [args.taskId]
+  sys.stderr.write('Tailing logs for tasks:\n')
+  for t in tasks:
+    sys.stderr.write('{0}\n'.format(t))
+  sys.stderr.write('ctrl+c to exit\n')
+  try:
+    threads = []
+    for task in tasks:
+      thread = LogStreamer(args, task)
+      threads += [thread]
```

Comment:
this is a huge nitpick i know, but just use `.append()` instead of concatenating the lists